lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1409,6 +1411,90 @@ void PrimTupleIndexOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
   });
 }
 
+//===----------------------------------------------------------------------===//
+// AtenSplitSizesOp
+//===----------------------------------------------------------------------===//
+
+void AtenSplitSizesOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
+                                                   MLIRContext *context) {
```

Comment:
This is not a canonicalization, but rather a lowering. This needs to go into a new pass called DecomposeOpsEarly.cpp which runs before shape refinement and ReduceOpVariants.

Reply:
Added the pass.